plugin/src/main/java/spp/jetbrains/sourcemarker/status/LogStatusBar.java

```diff
@@ -106,11 +124,15 @@ public Icon getIcon() {
         }).collect(Collectors.toList());
         lookup = text -> scopeVars.stream()
                 .filter(v -> {
-                    String var = substringAfterLast(" ", text.toLowerCase());
-                    return var.startsWith("$") && !var.substring(1).equals(v) && v.toLowerCase().contains(var.substring(1));
+                    String var = substringAfterLast(Constants.SPACE, text.toLowerCase());
```

Comment:
This logic looks to belong to VariableParser.

Reply:
refactored.